engine/src/hopeit/cli/server.py

```diff
@@ -26,12 +26,15 @@ def server():
 @click.option('--port', default='8020', help='TCP/IP port to listen.')
 @click.option('--path', help='POSIX complaint socket name.')
 @click.option('--start-streams', is_flag=True, default=False, help='Auto start reading stream events.')
-def run(config_files: str, api_file: str, host: str, port: int, path: str, start_streams: bool):
+@click.option('--groups', prompt='Comma-separated group label.',
```

Comment:
+1 OK to enable multiple groups at startup

DEFAULT groups must be `always` enabled in order to prevent the user to type `DEFAULT` all the time, also it is easy for events that need to be started all the time, just to have a DEFAULT group that is always started.